### PR TITLE
[WP#52241]update doc for release preparation and also after release

### PR DESCRIPTION
**Pull request by SagarGi**
### Description
Now that we have to show the change log in each release. We need to be sure that we add each change log during each release. So this PR adds documentation to add those change log during release. And also Adds documentation to generate change logs in github as well and not only in app store.

### Related WP
https://community.openproject.org/projects/nextcloud-integration/work_packages/52241/activity

### CODE REVIEW

**Review by individual-it — DISMISSED** [review]
also patch releases should have the changelog entries

File: README.md [hunk 1]
```diff
@@ -586,11 +586,15 @@ npm run watch
 5. Do QA and fixes of bugs in the release branch
 6. Update `.github/workflows/nighlty-ci-release-branch.yml` to run nightly on release branch
 7. Merge the release branch into the `master` branch, to get all good changes also into the current development
+8. Add change log in `CHANGELOG.md` with version to be released
```
Comment (nabim777) [inline README.md:589]:
```suggestion
8. Add change log in `CHANGELOG.md` with the version to be released
```

File: README.md [hunk 2]
```diff
@@ -586,11 +586,15 @@ npm run watch
...
+9. If any unreleased changes in `CHANGELOG.md` then add to the newly added change log
```
Comment (nabim777) [inline README.md:590]:
```suggestion
9. If any unreleased changes in `CHANGELOG.md` then add them to the newly added change log
```

File: README.md [hunk 3]
```diff
@@ -586,11 +586,15 @@ npm run watch
...
 
 ##### b. In case of a patch version
 
 1. On the release branch of the current minor version update the version in `appinfo/info.xml` (not needed for nightly builds)
 2. Merge the release branch into the `master` branch, to get all good changes also into the current development
+3. Add change log in `CHANGELOG.md` with version to be released
+4. If any unreleased changes in `CHANGELOG.md` then add to the newly added change log
```
Comment (nabim777) [inline README.md:597]:
```suggestion
4. If any unreleased changes in `CHANGELOG.md` then add them to the newly added change log
```

**Review by phil-davis — APPROVED** [review]

**Pull request merged**